Parsers/ASimAuditEvent/Parsers/vimAuditEventSentinelOne.yaml

```diff
@@ -0,0 +1,713 @@
+                                        "Low",
+                                        "Informational"
+                                    );
+        let ParsedActivitydata = union
+                groupsiteactivitydata,
+                machineactivitydata,
+                accountactivitydata,
+                useractivitydata,
+                rawotheractivitydata,
+                parsedotheractivitydata_eventtype,
+                parsedotheractivitydata_objectvalue
+            | where activityType_d !in(2030, 2036, 2037)
+            | lookup EventSeverityLookup on EventResult
+            | lookup EventSeverityLookup_activity on activityType_d
+            | where (eventresult == "*" or EventResult =~ eventresult)
+                and (array_length(eventtype_in) == 0 or EventType has_any (eventtype_in))
+                and (array_length(operation_has_any) == 0 or Operation has_any (operation_has_any))
+                and (array_length(newvalue_has_any) == 0 or NewValue has_any (newvalue_has_any))
+                and (array_length(object_has_any) == 0 or Object has_any (object_has_any))
+                and (array_length(srcipaddr_has_any_prefix) == 0 or has_any_ipv4_prefix(ipAddress, srcipaddr_has_any_prefix));
+        let UnParsedActivitydatawithThreat = union ParsedActivitydata, parsedotheractivitydata_severity
+            | where isnotempty(threatId_s)
+            | lookup Threatsdata on $left.threatId_s == $right.threatInfo_threatId_s
+            | where TimeGenerated1 >= TimeGenerated
+            | summarize arg_min(TimeGenerated1, *) by activityType_d, threatId_s, createdAt_t, TimeGenerated;
+        let undefineddata = UnParsedActivitydatawithThreat
+            | where threatInfo_confidenceLevel_s == "Undefined"
+            | lookup ThreatConfidenceLookup_undefined on threatInfo_analystVerdict_s;
+        let suspiciousdata = UnParsedActivitydatawithThreat
+            | where threatInfo_confidenceLevel_s == "suspicious"
+            | lookup ThreatConfidenceLookup_suspicious on threatInfo_analystVerdict_s;
+        let maliciousdata = UnParsedActivitydatawithThreat
+            | where threatInfo_confidenceLevel_s == "malicious"
+            | lookup ThreatConfidenceLookup_malicious on threatInfo_analystVerdict_s;
+        let ParsedActivitydatawithThreat = union undefineddata, suspiciousdata, maliciousdata
+            | extend
+                ThreatConfidence = coalesce(ThreatConfidence_undefined, ThreatConfidence_suspicious, ThreatConfidence_malicious),
+                AdditionalFields = bag_pack(
+                        "threatUpdatedAt",
+                        threatInfo_updatedAt_t,
+                        "threatAnalystVerdict",
+                        threatInfo_analystVerdict_s,
+                        "threatIncidentStatus",
+                        threatInfo_incidentStatus_s,
+                        "mitigationStatus",
+                        mitigationStatus_s
```

Comment:
@jayeshprajapaticrest 
Please include File info in additional fields where threat id is present
![image](https://github.com/Azure/Azure-Sentinel/assets/97222872/617239e3-5a36-472e-ae42-c4b163dfd599)

Reply:
It's already added in AdditionalField as part of DataFields_s after combining all the code blocks.